sale_sourced_by_line/migrations/8.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent - Miquel Raich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def populate_order_line_location_id(env):
+    env.cr.execute(
+        """
+        SELECT id, %s
+        FROM sale_order_line
+        """ % openupgrade.get_legacy_name('location_id')
```

Comment:
sale_order_line doesn't have location_id AFAIK

Reply:
in v7 :ghost: :rofl:

Reply:
yes

Reply:
yes...

![giphy](https://user-images.githubusercontent.com/23449160/32492416-7d221174-c3ba-11e7-9709-439b1f39dd86.gif)